code-challenges.js

```diff
@@ -21,48 +21,113 @@ const fruit2 = "banana"
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
-
-const temp = 42
+// I need to use a conditional statement to accept the variable "temp" and compare it to the boiling point (212). I'll use an if / else statement for this purpose.
```

Comment:
This is some beautiful pseudocode!!  Logic looks great and very well explained 👏